app/actions/remote/scheduled_post.test.ts

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import DatabaseManager from '@database/manager';
+import NetworkManager from '@managers/network_manager';
+import {getConfigValue} from '@queries/servers/system';
+
+import {deleteScheduledPost, fetchScheduledPosts} from './scheduled_post';
+
+const serverUrl = 'baseHandler.test.com';
+
+const scheduledPostsResponse: FetchScheduledPostsResponse = {
+    directChannels: [],
+    bar: [
+        {
+            id: 'scheduled_post_id',
+            channel_id: 'channel_id',
+            root_id: '',
+            message: 'test scheduled post',
+            scheduled_at: 123,
+            user_id: 'user_id',
+            processed_at: 0,
+            update_at: 456,
+            error_code: '',
+        },
+        {
+            id: 'scheduled_post_id_2',
+            channel_id: 'channel_id',
+            root_id: '',
+            message: 'test scheduled post 2',
+            scheduled_at: 123,
+            user_id: 'user_id',
+            processed_at: 0,
+            update_at: 456,
+            error_code: '',
+        },
+    ],
+};
+
+const throwFunc = () => {
+    throw Error('error');
+};
+
+const mockClient = {
+    getScheduledPostsForTeam: jest.fn(() => Promise.resolve({...scheduledPostsResponse})),
+    deleteScheduledPost: jest.fn((scheduledPostId) => {
+        return Promise.resolve(scheduledPostsResponse.bar.find((post) => post.id === scheduledPostId));
+    }),
+};
+jest.mock('@queries/servers/system', () => ({
+    getConfigValue: jest.fn(),
+}));
+
+const mockedGetConfigValue = jest.mocked(getConfigValue);
+
+beforeAll(() => {
+    // eslint-disable-next-line
+    // @ts-ignore
+    NetworkManager.getClient = () => mockClient;
+});
+
+beforeEach(async () => {
+    await DatabaseManager.init([serverUrl]);
+});
+
+afterEach(async () => {
+    await DatabaseManager.destroyServerDatabase(serverUrl);
+});
+
+describe('fetch and delete scheduled posts', () => {
```

Comment:
Use one describe for each function. That way the name of the test also can be simplified without so much repetition.

Reply:
Updated the file.